lib/src/components/layout/accordion.dart

```diff
@@ -20,39 +20,149 @@ class _AccordionState extends State<Accordion> {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     final scaling = theme.scaling;
+    final accTheme = ComponentTheme.maybeOf<AccordionTheme>(context) ??
+        const AccordionTheme();
     return Data.inherit(
         data: this,
         child: IntrinsicWidth(
-          child: Column(
-              mainAxisSize: MainAxisSize.min,
-              crossAxisAlignment: CrossAxisAlignment.stretch,
-              children: [
-                ...join(
-                    widget.items,
-                    ShadcnAnimatedContainer(
-                      duration: kDefaultDuration,
-                      color: theme.colorScheme.muted,
-                      height: 1 * scaling,
-                    )),
-                const Divider(),
-              ]),
+          child: ComponentTheme(
```

Comment:
Do not use ComponentTheme here. ComponentTheme is a holder for the theme itself that later be used by the widget/component to override some of the default values